test/aqua/test_grad.py

```diff
@@ -0,0 +1,669 @@
+# This code is part of Qiskit.
```

Comment:
As the tests around logic in the operators are in an operators folder I think perhaps it would be more consistent if this were moved there and maybe renamed to `test_gradients` since its still short enough but perhaps more easily equatable to the 'gradients' components with it spelled out in full.